src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -184,7 +184,13 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
                 }
             },
             set entrezGeneId(e:number|undefined) {
-                this._entrezGeneId = e;
+                runInAction(()=>{
+                    if (e !== this._entrezGeneId) {
```

Comment:
my guess about what's happening here is, if you change gene, you want to clear the other selections?   if that's case, maybe what we need is "clear" routine, which gets called when you change gene.  this would encapsulate the logic for clearing, make it reusable, more readable.   i know we're working fast now but this might be worthy of test?

Reply:
I see what you're saying. but I don't think this would be reused and its too simple to test. do you think a comment be enough?

Reply:
sure